src/OrchardCore.Modules/OrchardCore.Users/Controllers/AuthenticatorAppController.cs

```diff
@@ -59,7 +59,6 @@ public class AuthenticatorAppController : TwoFactorAuthenticationBaseController
         _shellSettings = shellSettings;
     }
 
-    [Admin("Authenticator/Configure/App", "ConfigureAuthenticatorApp")]
```

Comment:
Why not keep the routes here with attribute routing? It's simpler. Also for `TwoFactorAuthenticationController`, `EmailAuthenticatorController`, and `SmsAuthenticatorController`.

Reply:
because we can't use the `AdminAttribute` since the user may not have the AccessAdminPanel permission. which is the issue this PR addresses.

Reply:
I'm talking about standard [ASP.NET Core attribute routing](https://learn.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-8.0#ar6) with the `[Route]` attribute.

Reply:
I rather stick to what we have been using in this project everywhere else.

Reply:
You changed a lot of stuff unrelated to the code task at hand here anyway, so I don't see why keeping the routes in attributes, as they were, would be an issue over adding them as new configs in the `Startup`s. But OK.